src/Util/getTypeName.js

```diff
@@ -0,0 +1,18 @@
+const ONE_ROOM = 'ONE_ROOM';
+const MULTI_ROOM = 'MULTI_ROOM';
+const OFFICETEL = 'OFFICETEL';
+
+export const MAX_NUMBER = 1000000000000000000;
```

Comment:
```suggestion
export const MAX_NUMBER = 1_000_000_000_000_000_000;
```
- 최대값이 너무 크게 잡힌거 아닌가 하는 느낌이 드네요
- 언더스코어를 숫자 구분자로 사용할 수 있습니다

Reply:
금액을 억단위로 축소하였고 _를 통해 금액을 좀더 확인하기 편하도록 변경하였습니다.
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Lexical_grammar 
https://writingjavascript.com/what-are-numeric-separators